process/process.go

```diff
@@ -672,6 +672,14 @@ func (p *Process) setEnv() {
 
 func (p *Process) setDir() {
 	dir := p.Config().Directory
+
+	_, err := os.Stat(dir)
+	if err != nil {
+		errString := fmt.Sprintf("The directory %s does not exist", dir)
+		p.log.Error(errString)
```

Comment:
Tiny bit of stylistic change here since `zap` provides structured logging:

```
p.log.Error("Directory does not exist.", zap.String("directory", dir))
```